testsuite/tests/contrib/lkt_semantic/array_type_concrete/test.out

```diff
@@ -0,0 +1,96 @@
+Id   <RefId "Int" test.lkt:1:8-1:11>
```

Comment:
I believe this file contains “garbade” (i.e. does not correspond to the actual test). I suggest make it an empty file to avoid confusion.

Reply:
This is the output of lkt_toolbox on the test.lkt file.

Reply:
Yes, but that’s not the expected output: for instance nodes from the prelude have slocs (they shouln’t in baselines), line numbers are slightly off and most importantly, this file makes it feel like the typing errors are expected (because of the lack of type inference from array literal elements). And if you update the test.out to be the current output (with the `Resolving test.lkt` header), you’ll even get an UOK (unexpected success) test instead of an XFAIL one (expected failure), making one think that the test actually passes, whereas it should be failing (waiting for the TODO to be implemented).

Reply:
OK, I'll empty this file then!